<!-- -->
<launch>

	<arg name="ars_sim_obstacles_detector_node_name" 	default="ars_sim_obstacles_detector_node" />
	<arg name="screen" 	default="screen" />


  <node name="$(arg ars_sim_obstacles_detector_node_name)" pkg="ars_sim_obstacles_detector" type="ars_sim_obstacles_detector_ros_node.py" output="$(arg screen)" >


  </node>


</launch>
